views/index.hbs

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Essence - Ecommerce Candle Shop W/ DB</title>
+
+    <!-- font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=Work+Sans:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+</head>
```

Comment:
Hey Judy, this is my bad for not explaining in class, but you don't need to put this part in `index.hbs`. Instead, you can link all your font/stylesheets/javascript files in `layout.hbs` because `layout.hbs` is actually a wrapper around all your other view files.